src/components/top-nav-default/TopNavDefault.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import { STopNavDefaultTitle } from "./styles/STopNavDefaultTitle";
+import { STopNavDefaultDivider } from "./styles/STopNavDefaultDivider";
+import { TopNavDefault } from "./TopNavDefault";
+import { Button } from "../button/Button";
+import { Tab } from "../tab/Tab";
+import { IconChevronDown, IconStarOutlined } from "../../icons";
+
+const tabs = [
+  { name: "Activated", id: 0 },
+  { name: "Default", id: 1 },
+  { name: "Default", id: 2 },
+  { name: "Default", id: 3 },
+  { name: "Default", id: 4 },
+  { name: "Default", id: 5 },
+  { name: "Default", id: 6 },
+];
+
+export default {
+  title: `Components/Navigation/TopNavDefault`,
+  component: TopNavDefaultDemo,
+  argTypes: {},
+};
+
+function TopNavDefaultDemo({ withButton = false, withTabs = false, withIcon = false }) {
+  const [selectedTab, setSelectedTab] = useState(0);
+  return (
+    <TopNavDefault>
+      <div style={{ display: "flex", padding: "0 16px", alignItems: "center", flex: 1 }}>
+        <STopNavDefaultTitle>
```

Comment:
@nicolasshiken please make these exported components, SComponents are internal impl that should not be exposed.